packages/gasket-cli/src/scaffold/actions/global-prompts.js

```diff
@@ -31,7 +31,7 @@ async function chooseAppDescription(context) {
  * @returns {Promise} promise
  */
 async function choosePackageManager(context) {
-  const packageManager =
+  const packageManager = context.package ||
```

Comment:
Similar to the above comment. Don' introduce a new context property, just tune your tests to utilize the existing `context.packageManager`.